TGC/decisions/FranceExtra.txt

```diff
@@ -568,4 +568,44 @@ political_decisions = {
 		}
 		ai_will_do = { factor = 1 }
 	}
-}
\ No newline at end of file
+	propose_talleyrand_again = {
+		potential = {
+			OR = {
+				tag = FRA
+				tag = BOR
+				}
+			NET = { NOT = { has_country_flag = signed_treaty_of_london } }
+			NOT = { has_country_flag = talleyrand_proposed }
+			}
+
+		allow = {
+			war = no
+			NOT = {
+				relation = { who = ENG value = 0 }
+				relation = { who = PRU value = 0 }
+			 }
+		}
+
+		effect = {
+			ENG = { country_event = 36719 }
+			badboy = 3
+			FRA = { set_country_flag = talleyrand_proposed }
```

Comment:
Not sure how sending events to nonexistent countries works, maybe just do random_country instead?

Reply:
in my tests it didn't seem to break anything, but sure I can change it to a random_county effect